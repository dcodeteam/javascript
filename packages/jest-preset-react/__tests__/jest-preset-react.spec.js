"use strict";

const { initJestPresetTest } = require("test-tools");

initJestPresetTest("jest-preset-react", require.resolve("../jest-preset"));
