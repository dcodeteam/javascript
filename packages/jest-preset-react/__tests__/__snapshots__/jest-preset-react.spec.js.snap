// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-preset-react should match snapshot 1`] = `
Object {
  "collectCoverageFrom": Array [
    "src/**/*.ts?(x)",
    "!src/**/__docs__/**/*",
    "!src/**/__tests__/**/*",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 100,
      "functions": 100,
      "lines": 100,
      "statements": 100,
    },
  },
  "moduleFileExtensions": Array [
    "ts",
    "tsx",
    "js",
  ],
  "snapshotSerializers": Array [
    "enzyme-to-json/serializer",
  ],
  "testEnvironment": "jsdom",
  "testMatch": Array [
    "**/*.spec.ts?(x)",
  ],
  "transform": Object {
    "^.+\\\\.(ts|tsx)$": "babel-jest",
  },
}
`;
